util/cmake/SeqAnContribs.cmake

```diff
@@ -69,6 +69,12 @@ if (WIN32)
 		  set (SEQAN_CONTRIB_BASE "C:\\${_SEQAN_CONTRIB_DIR}-${CONTRIB_ARCH}")
 		endif ()
 	  endif (NOT DEFINED SEQAN_CONTRIB_BASE)
+    # Try to find in specific environment variable
```

Comment:
You could iterate over all possible directory paths.

In this code we always do the same:
* If SEQAN_CONTRIB_BASE is not known, yet
   * Look if the new path is existing
   * If not repeat

This is basically a for loop over all directories.

That can be  simplified:
```cmake
set(_paths "$ENV{ProgramW6432}" "$ENV{ProgramFiles}" "C:\\" "$ENV{SEQAN_WIN_CONTRIB_DIRECTORY}")

foreach(path $_paths)
        set(path "$path/${_SEQAN_CONTRIB_DIR}-${CONTRIB_ARCH}"
        if (IS_DIRECTORY "$path")
            set (SEQAN_CONTRIB_BASE "$path")
            break ()
        endif ()
endforeach()
```
(Might be no valid cmake syntax)